spec/model/vm_host_spec.rb

```diff
@@ -82,6 +82,35 @@
     vh.download_boot_image("my-image", custom_url: "https://example.com/my-image.raw", version: "20230303")
   end
 
+  it "has a shortcut to download a new firmware for x64" do
+    vh.id = "46683a25-acb1-4371-afe9-d39f303e44b4"
+    vh.arch = "x64"
+    expect(Strand).to receive(:create) do |args|
+      expect(args[:prog]).to eq("DownloadFirmware")
+      expect(args[:stack]).to eq([subject_id: vh.id, version: "202405", sha256: "sha-1"])
+    end
+    vh.download_firmware(version_x64: "202405", version_arm64: "202406", sha256_x64: "sha-1", sha256_arm64: "sha-2")
```

Comment:
Do we expect/want caller to pass version info for the non-used arch as well?

Reply:
We don't.